### PR TITLE
fixed seeding && fixed waittime on TA's

### PR DESCRIPTION
**Pull request by NEUDitao**
gentle reminder that any change you make in TAQueueCard and TAQueueList must be done in student as well and vice versa

### CODE REVIEW

File: packages/app/components/Queue/TAQueueCard.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export default function TAQueueCard({
   question,
   onOpen,
 }: TAQueueCardProps): ReactElement {
+  const waitTime = now.getMinutes() - question.createdAt.getMinutes();
```
Comment (liustanley) [inline packages/app/components/Queue/TAQueueCard.tsx:26]: i think you forgot to copy over `now`

Comment (willstenzel) [reply packages/app/components/Queue/TAQueueCard.tsx:26]:
```suggestion
  const now = new Date();
  const waitTime = now.getMinutes() - question.createdAt.getMinutes();
```

Comment (NEUDitao) [reply packages/app/components/Queue/TAQueueCard.tsx]: i did and then committed but never pushed -- there's a new bug there you might wanna check out :P

File: packages/server/src/question/question.entity.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export class QuestionModel extends BaseEntity {
   @Exclude()
   taHelpedId: number;
 
-  @CreateDateColumn()
+  @Column()
```
Comment (dajinchu) [inline packages/server/src/question/question.entity.ts:48]: What's the reason for this?

Comment (liustanley) [reply packages/server/src/question/question.entity.ts:48]: CreateDateColumn wasn't correctly storing the createdAt date, it did something funky with the UTC conversion and actually messed up our times. I changed it to a normal Column and pass in `new Date()` when creating a question which works fine, but not sure if we should investigate this further.

**Comment by liustanley** [comment]
https://github.com/typeorm/typeorm/issues/3286

**Pull request merged**